heartbeat/rabbitmq-server-ha

```diff
@@ -702,14 +702,20 @@ rabbit_node_name() {
 rmq_setup_env() {
     local H
     local dir
+    local name
     H="$(get_hostname)"
     export RABBITMQ_NODENAME=$(rabbit_node_name $H)
     if [ "$OCF_RESKEY_node_port" != "$OCF_RESKEY_node_port_default" ]; then
         export RABBITMQ_NODE_PORT=$OCF_RESKEY_node_port
     fi
     export RABBITMQ_PID_FILE=$OCF_RESKEY_pid_file
     MNESIA_FILES="${OCF_RESKEY_mnesia_base}/$(rabbit_node_name $H)"
-    export RABBITMQ_SERVER_START_ARGS="${RABBITMQ_SERVER_START_ARGS} -mnesia dir \"${MNESIA_FILES}\" -sname $(rabbit_node_name $H)"
+    if [ "${OCF_RESKEY_use_fqdn}" = 'false' ] ; then
```

Comment:
`if ! ocf_is_true "${OCF_RESKEY_use_fqdn}"` is better, as users may use other true/false values as well.

Reply:
oh thanks! I'll fix that. Missed this comment somehow.